uftrace.c

```diff
@@ -1057,6 +1077,7 @@ int main(int argc, char *argv[])
 
 	switch (opts.mode) {
 	case UFTRACE_MODE_RECORD:
+		check_shmem_freespace();
```

Comment:
To keep consistency I'd like to leave call to `command_record()` here.  Would you move this **into** the `command_record()`?